src/core/Akka/IO/UdpConnectedManager.cs

```diff
@@ -42,22 +40,7 @@ protected override bool Receive(object message)
                         Context.ActorOf(Props.Create(() => new UdpConnection(_udpConn, commander, connect)));
                         return true;
                     }
-                case DeadLetter dl when dl.Message is UdpConnected.SocketCompleted completed:
```

Comment:
No need for this anymore, because SocketAsyncEventArgs will never be embedded in messages.